src/sidebar/FeedList/Folder.tsx

```diff
@@ -107,50 +118,94 @@ const Folder = (props: Props) => {
         handleDragLeave,
         handleDrop,
         handleDragEnd,
-    } = useDragDropNode(props.nodeMeta);
+    } = useDragDropNode(nodeMeta);
 
     if (!props.showTitle) {
         return <Fragment>{props.children}</Fragment>;
     }
 
+    const handleMouseDown = (e: React.MouseEvent<HTMLDivElement, MouseEvent>) => {
+        if (e.button !== MouseEventButton.leftMouseButton && e.button !== MouseEventButton.rightMouseButton) {
+            return;
+        }
+
+        if (e.button === MouseEventButton.leftMouseButton) {
+            // TODO change expanded on mouseUp to prevent it before drag
+            setExpanded(!expanded);
+        }
+
+        if (!focus) {
+            setFocus(true);
+        }
+
+        if (selectedId !== nodeMeta.nodeId) {
+            dispatch(feedsSlice.actions.select(nodeMeta));
+        }
+    };
+
+    const handleBlur = () => {
+        if (focus) {
+            setFocus(false);
+        }
+    };
+
+    // TODO auslagern nach ContextMenuTarget
```

Comment:
resolve this TODO

Reply:
https://github.com/dermeck/feeds-sidebar/pull/176/commits/b2d5be374f3f568dbef32701417a17740687f47d